tests/test_vec_envs.py

```diff
@@ -0,0 +1,36 @@
+import gym
+from stable_baselines.common.vec_env.dummy_vec_env import DummyVecEnv
+from stable_baselines.common.vec_env.subproc_vec_env import SubprocVecEnv
+
+ENV_ID = 'BreakoutNoFrameskip-v4'
```

Comment:
I would avoid relying on atari env. I suggest you to create a dummy gym env that has custom method, so you can test passing arguments and keyword arguments ;).